web/src/scheduled_messages.js

```diff
@@ -315,3 +317,29 @@ export function initialize(scheduled_messages_params) {
         e.stopPropagation();
     });
 }
+
+// exporting for testability
+export const _test_should_update_send_later_options = should_update_send_later_options;
```

Comment:
You should able to test `should_update_send_later_options()` directly; there's no need I can see to export the function reference as a separate variable.

Reply:
Sure!